apex/contrib/test/openfold_triton/test_sync_triton_auto_tune_cache_across_gpus.py

```diff
@@ -0,0 +1,95 @@
+import os
+import torch
+import torch.distributed as dist
+from torch.testing._internal.common_distributed import (
+    MultiProcessTestCase,
+    requires_nccl,
+    skip_if_lt_x_gpu,
+    run_tests,
+)
+from apex.contrib.openfold_triton import (
+    LayerNormSmallShapeOptImpl,
+    sync_triton_auto_tune_cache_across_gpus,
+    _tuneable_triton_kernels,
+)
+
+class SyncTritonAutoTuneCacheTest(MultiProcessTestCase):
+    device_type = "cuda"
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+    def setUp(self) -> None:
+        super().setUp()
+        self._spawn_processes()
+
+    def tearDown(self) -> None:
+        torch.cuda.synchronize()
+        torch.cuda.empty_cache()
+        super().tearDown()
+
+    @property
+    def world_size(self) -> int:
+        return min(torch.cuda.device_count(), 2)
+
+    @property
+    def init_method(self):
+        return f"{common_utils.FILE_SCHEMA}{self.file_name}"
+
+    @property
+    def destroy_pg_upon_exit(self) -> bool:
+        return True
+
+    def _create_process_group_nccl(self):
+        def maybe_export(env, val):
+            assert(type(env) == str)
+            assert(type(val) == str)
```

Comment:
nit-picking
```suggestion
            assert type(env) == str
            assert type(val) == str
```
or even it'd be better to raise an exception